spec/models/user_spec.rb

```diff
@@ -21,8 +37,10 @@
   end
 
   it 'sends an email' do
-    #byebug
-    expect { donor.send_instructions }
-      .to change { ActionMailer::Base.deliveries.count }.by(1)
+   # ActionMailer::Base.deliveries.clear
+    donor.send(:send_registered_email)
```

Comment:
It was a different test originally to test basic mailer functionality. I could go back and loop over all the emails once they've been queued and check them all